overlord/snapstate/handlers.go

```diff
@@ -1383,6 +1407,13 @@ func (m *SnapManager) doDiscardSnap(t *state.Task, _ *tomb.Tomb) error {
 		if err := m.backend.RemoveSnapDir(snapsup.placeInfo(), otherInstances); err != nil {
 			return fmt.Errorf("cannot remove snap directory: %v", err)
 		}
+
+		// try to remove the auxiliary store info
+		if err := discardAuxStoreInfo(snapsup.SideInfo.SnapID); err != nil {
```

Comment:
<strike>Is this tested in handlers_discard_test.go?</strike> - I see this is tested in TestRemoveRunThrough